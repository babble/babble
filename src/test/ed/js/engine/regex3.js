

var p = /(\d)/g

var a = "a";
var b = "1";

var c = "1a2b3c";
while ( r = p.exec( c ) ){
    print( r );
}

x = "1a2";
y = "1b";
print( p.exec( x ) );
print( p.lastIndex );

print( p.test( y ) );
print( p.lastIndex );

print( p.exec( y ) );
print( p.lastIndex );

