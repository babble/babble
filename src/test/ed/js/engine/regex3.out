1,1
2,2
3,3
1,1
1
false
0
1,1
1
